### ISSUE TITLE
[Docencia] Tercer curso

### ISSUE THREAD

**Issue by JJavier98** [OpenETSIIT/openetsiit.github.io#1]
## Describe el problema
Las asignaturas de tercer año son todas en horario de tarde mientras que primero, segundo y cuarto se desarrollan por la mañana.

## Sugiere posibles soluciones al mismo
Dar utilidad al edificio específico para másteres y así poder repartir mejor los horarios y aulas en el edificio principal de la facultad o terminar de construir el triángulo que conforma el edificio principal para tener un aulario más amplio y poder tener un horario más flexible, poder dar más clases, mayor variedad de temarios, etc

**Comment by JJ** [comment]
Primero y segundo creo que tienen clases tanto por la mañana como por la tarde, ¿no?
El problema de espacio, por otro lado, está claro.

**Comment by JJavier98** [comment]
Sí, creo que hay opción de escoger clases por la tarde en primero y segundo, pero ya en cuarto no.
El caso es que el cambiar de horario (ya sea en tercero por la tarde o en cuarto por la mañana) supone en muchos casos cambios de horario en el trabajo o el tener que dejar actividades externas a la universidad, que en mi opinión, son cosas que no deberían verse afectadas de manera obligada.
